Knittery/Network/QueryBuilder.swift

```diff
@@ -45,24 +56,18 @@ class QueryBuilder {
     static private let catQuery = "$"
```

Comment:
I would consider renaming `catQuery` to something like `concatenation`

Reply:
Name changed to concat for clarity